### PR TITLE
Slider: Pass 0 value if slider reaches it

### PR DESCRIPTION
**Pull request by atomiomi**
If changed value is equal to zero then the last changed value is passed to `ui` object. This is due to checking in `_uiHash` method which defines whether the new value was passed or not.

Test link: https://jsfiddle.net/Lhqryk47/
1. Drag the handle on the right side
2. Value must be between 1 and 100
3. Now drag the handle on the minimum value (zero)
4. Passed value will be 1 or greater it depends on drag speed

### CODE REVIEW

File: ui/slider.js [hunk 1]
```diff
@@ -299,13 +299,13 @@ return $.widget( "ui.slider", $.ui.mouse, {
...
 	_uiHash: function( index, value, values ) {
 		var uiHash = {
-			handle: this.handles[ index ],
-			handleIndex: index,
-			value: value || this.value()
-		};
+				handle: this.handles[ index ],
+				handleIndex: index,
+				value: typeof value !== "undefined" ? value : this.value()
+			};
```
Comment (jzaefferer) [inline ui/slider.js]: Please undo this indent change, on this line and the three above.

**Comment by atomiomi** [comment]
Indents are corrected.

**Comment by jzaefferer** [comment]
Looks good. I'll let @scottgonzalez  review and merge.

**Comment by scottgonzalez** [comment]
Thanks. Filed the bug as http://bugs.jqueryui.com/ticket/12196, replaced `typeof value !== "undefined"` with `value !== undefined` for consistency, and merged into master.

**Pull request closed without merging**